Changelog.md

```diff
@@ -1,13 +1,27 @@
 # Unreleased
 
+# 5.4.2
+
+### .NET8
+* Build with .NET8 sdk and run AspNetCore tests with .NET8 by @Daniel-Svensson in https://github.com/OpenRIAServices/OpenRiaServices/pull/463
+* Run most client, server and code generation tests with .NET8 @SandstromErik in https://github.com/OpenRIAServices/OpenRiaServices/pull/458
+
+### Code generation
+* Do not use attributes reserved for the compiler in code generation. Will make it possible to use the `required` modifier for fields and properties @SandstromErik in https://github.com/OpenRIAServices/OpenRiaServices/pull/472
+
 ### AspNetCore updates
```

Comment:
Consider making this a top level header "#" with "aspnetcore hosting 1.0 *needs some moving)


Pr maybe better change above header to "5.4.2 / AspNetCore 1.0.0" or similar